io.github.aggalex.Wineglass.yaml

```diff
@@ -0,0 +1,171 @@
+app-id: io.github.aggalex.Wineglass
+base-version: stable-22.08
+base: org.winehq.Wine
+runtime-version: &runtime-version '22.08'
```

Comment:
```suggestion
runtime-version: &runtime-version '23.08'
```

Reply:
As i [mentioned](https://github.com/flathub/flathub/pull/4507#issuecomment-1716684608) earlier i plan to upgrade on 23.08 a little bit later when DXVK was built for 23.08.